### PR TITLE
Update to latest versions

### PR DESCRIPTION
**Pull request by patriknw**
Just patch version updates aside from akka-grpc.
We should release 0.2.0 or why not 1.0.0 from this?

### CODE REVIEW

**Comment by patriknw** [comment]
When this has been released we should update the Platform Guide.

File: project/Dependencies.scala [hunk 1]
```diff
@@ -81,21 +81,22 @@ object Dependencies {
...
 
   val akkaGrpc = Seq(
     // maven plugin brings in Scala dependencies that conflict with current crossVersion of BOM
     // `Modules were resolved with conflicting cross-version suffixes in ProjectRef(uri("file:/home/seglo/source/akka-platform-guide/akka-platform-dependencies/"), "akka-platform-dependencies") ...`
     // maven plugin versions can't be set with BOM/dependencyManagement
     //("com.lightbend.akka.grpc" % "akka-grpc-maven-plugin" % AkkaGrpc).withExclusions(Vector(ExclusionRule.everything)),
-    "com.lightbend.akka.grpc" %% "akka-grpc-runtime" % AkkaGrpc,
-    "com.lightbend.akka.grpc" %% "akka-grpc-codegen" % AkkaGrpc
+    "com.lightbend.akka.grpc" %% "akka-grpc-runtime" % AkkaGrpc
+    //"com.lightbend.akka.grpc" %% "akka-grpc-codegen" % AkkaGrpc
```
Comment (patriknw) [inline project/Dependencies.scala:99]:
```suggestion
    // TODO can akka-grpc-codegen be removed from bom?
    // not sure why, but akka-grpc-codegen is only released for Scala 2.12
    // (and I don't think it's supposed to be used via bom anyway?)
    //"com.lightbend.akka.grpc" %% "akka-grpc-codegen" % AkkaGrpc
```

[the remark this answers is not in the document]
Comment (patriknw) [reply project/Dependencies.scala:99]: @johanandren added a TODO comment as suggestion here...

Maybe @raboof knows and we can remove akka-grpc-codegen from this right away?

Comment (raboof) [reply project/Dependencies.scala:99]: Publishing the codegen for 2.13 is definitely something we want, but broke when we switched to publishing to sonatype with sbt-ci-release (https://github.com/akka/akka-grpc/issues/1400).

I agree it doesn't make too much sense to have it in the bom, though.

Comment (patriknw) [reply project/Dependencies.scala:99]: Thanks for confirming, I have removed it

**Review by johanandren — APPROVED** [review]

**Pull request merged**